Source/WTF/wtf/JSONValues.h

```diff
@@ -52,7 +53,7 @@ class Object;
 class ObjectBase;
 
 // FIXME: unify this JSON parser with JSONParse in JavaScriptCore.
-class WTF_EXPORT_PRIVATE Value : public RefCounted<Value> {
+class WTF_EXPORT_PRIVATE Value : public RefCounted<Value>, public NoVirtualDestructorBase {
```

Comment:
For what it's worth, I don't like the `NoVirtualDestructorBase` name but I assume it's hardcoded in the static analysis? Ideally the name would be something like "SuppressVirtualDestructorChecks" or "SuppressVirtualDestructorStaticAnalysis". But at least we have a comment next to `NoVirtualDestructorBase` explaining that it is about static analysis..

Reply:
yeah, it's hard-coded.